android-mode.el

```diff
@@ -235,11 +235,11 @@ environment value otherwise the `android-mode-sdk-dir' variable."
   "Run COMMAND named NAME with ARGS unless it's already running."
   (and (not (cl-find (intern name) android-exclusive-processes))
        (set-process-sentinel (apply #'start-process-shell-command name name
-                                    (concat command
-                                            " "
-                                            (mapconcat #'shell-quote-argument
-                                                       args
-                                                       " ")))
+                                    (list (concat command
+                                                  " "
+                                                  (mapconcat #'shell-quote-argument
+                                                             args
+                                                             " "))))
```

Comment:
We should probably get rid of `apply` instead:

```
       (set-process-sentinel (start-process-shell-command name name
                                                          (concat command
                                                                  " "
                                                                  (mapconcat #'shell-quote-argument
                                                                             args
                                                                             " ")))
```

Can you try if that works for you?  I currently don't have a working android setup to test with at hand.

Reply:
This works!